app/src/main/java/org/groundplatform/android/model/submission/InstructionTaskData.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.groundplatform.android.model.submission
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class InstructionTaskData(val text: String) : TaskData {
+  override fun isEmpty(): Boolean = text.trim { it <= ' ' }.isEmpty()
```

Comment:
Let's rethink what it means for this task. We use this method to check if the task has any value or not. Given this is a read-only task, let's ensure we are doing the right thing here. Also, we should add a unit test for the same